scripts/globals/mobskills/Numbing_Noise.lua

```diff
@@ -12,6 +12,19 @@ require("/scripts/globals/status");
 require("/scripts/globals/monstertpmoves");
 
 ---------------------------------------------
+-- For Race Runner
+local path = {
```

Comment:
It's supposed to use this move and start running again. I am not sure how else to do it. Maybe in onMobWeaponSkill?